portfolio/src/main/webapp/charts_script.js

```diff
@@ -0,0 +1,31 @@
+google.charts.load('current', {'packages':['corechart']});
+google.charts.setOnLoadCallback(drawStaticPieChart);
+ 
+/** Creates a chart and adds it to the page. */
+function drawStaticPieChart() {
+  var data = google.visualization.arrayToDataTable([
```

Comment:
Would `const` work here? Otherwise let's use `let`

https://google.github.io/styleguide/jsguide.html#features-use-const-and-let

Reply:
Done. I should double check the templates that I use.